src/index.css

```diff
@@ -1,13 +1,127 @@
+
+html { box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+/* Basic styles */
+.wrapper {
+    max-width: 1260px;
+    width: 90%;
+    margin: 0 auto;
+}
+
+h1, h2, h3, p {
+    margin: 0;
+}
+
+html {
+    font-size: 125%;
+}
+
+/* Header Styles */
+header {
+    background-image: url(./Assets/headerBackground.jpg);
+    height: 100vh;
+    background-size: cover;
+    background-position: center;
+    opacity: 0.8;
+
+}
+
+h1 {
+    color: white;
+    font-family: 'Rubik Distressed', cursive;
+    font-style: italic;
+    text-decoration: underline;
+    font-size: 3.5rem;
+    padding: 50px;
+    text-align: center;
+}
+
+h2 {
+    font-family: 'Rubik Distressed', cursive;
+    font-size: 2rem;
+}
+
+.statsHeader {
+    font-size: 2rem;
+    padding-top: 50px;
+    text-decoration: underline;
+}
+
+ul {
+    list-style: none;
+    margin: 0;
+    padding:0;
+}
+
+/* Form */
+
+form {
+    margin: 50px 0 50px 0;
+}
+
+/* Stats Section */
+
+.statsSection {
```

Comment:
just a gentle reminder the commented out style is still there if you're not using it anymore